pubsub_nats_options.go

```diff
@@ -76,10 +96,9 @@ type natsSubscribeConfig struct {
 }
 
 type natsPublishConfig struct {
-	ctx            context.Context
-	replyValidator func(msg *nats.Msg) error
-	useRequestMode bool
-	responseCh     chan *Publication
+	ctx         context.Context
+	requestMode requestMode
+	responseCh  chan *Publication
 }
 
 // NATSOptSubscribeQueue sets the NATS subscriber queue group.
```

Comment:
why is this commented out? this is widely used

Reply:
I haven't, it's still there. 5 lines below to be exact :D